src/service.rs

```diff
@@ -111,15 +111,15 @@ pub enum ServiceId {
     #[from]
     Peer(NodeAddr),
 
-    #[display("swap<{0:#x}>")]
```

Comment:
this was one of the few places that was printing the swapd correctly, are you sure about this?

Reply:
The swap if can be printed with its full version in the logs one it start if you want, and we can access it with cli ls

Reply:
Done in 83619a737880eefab230b74327a8b0d7f5ae8fcd